spring-cloud-starter-stream-processor-tensorflow/src/main/java/org/springframework/cloud/stream/app/tensorflow/processor/TensorflowProcessorProperties.java

```diff
@@ -37,24 +39,46 @@
 	private Resource modelLocation;
 
 	/**
-	 * The graph model output name. Also know as fetch.
+	 * The graph model output. Name of TensorFlow operation to fetch the output Tensors from.
 	 */
 	private String modelFetchName;
 
 	/**
-	 * The graph model index withing the fetch Tensor.
+	 * Fetched operation (modelFetchName) returns a list of Tensors. The modelFetchIndex specifies which
+	 * Tensor from that list to use as an output.
 	 */
 	private int modelFetchIndex = 0;
 
 	/**
-	 * Specifies where to obtain the input data from. By default it looks
-	 * at the {@link org.springframework.messaging.Message}'s payload.
-	 * Instead one can obtain the input value from a payload {@link org.springframework.tuple.Tuple} like this:
-	 * 'tensorflow.inputExpression=payload.myInTupleName', where myInTupleName is a Tuple key.
-	 * To obtain input date from the message headers use expression like this:
-	 * 'tensorflow.inputExpression=headers[myHeaderName]', where is the name of the header that contains the input data.
+	 * Specifies where to obtain the input data from. When empty defaults to {@link org.springframework.messaging.Message}'s payload.
+	 * To obtain an input from a {@link org.springframework.tuple.Tuple} set:
+	 * 'tensorflow.expression=payload.myInTupleName', where myInTupleName is a Tuple key.
+	 * To obtain input date from a message header use:
+	 * 'tensorflow.expression=headers[myHeaderName]', where is the name of the header that contains the input data.
 	 */
-	private String inputExpression = "payload";
+	private Expression expression;
+
+	/**
+	 * Specifies how the outbound data is carried and whether the input message is mirrored in the output or discarded.
+	 *
+	 * The {@link OutputMode#payload} mode (default) stores the output data in the outbound message payload. The
```

Comment:
Correct, that's the way the boot configuration plugin works. Javadoc for those props should be limited to the strict minimum, short, no formatting sentence.
If needed, you can always add an additional, __regular__ comment